README.md

```diff
@@ -41,9 +41,18 @@ Add a new stanza to the "rule_engines" array within `server_config.json`:
     "plugin_specific_configuration" : {
          "amqp_location" : "ANONYMOUS@localhost:5672",
          "amqp_topic" : "audit_messages",
-         "pep_regex_to_match" : "pep_.+"
+         "pep_regex_to_match" : "audit_pep_(api|resource)_.*"
      }
-}
+},
+```
+
+Add the "audit_" namespace to the "rule_engine_namespaces" as well:
+
+```diff
+    "rule_engine_namespaces": [
+        "",
++        "audit_"
```

Comment:
Seems weird to show a diff in the README. I don't think it's necessary given L49 and the fact that most deployments will likely only contain one element in the `rule_engine_namespaces` stanza.

Reply:
well, right - but how would we convey to add?    oh, you're saying, just the line 49 text is sufficient?

also, initial point still stands... do we even need this "audit_" anymore?   does the dashboard need to look for different messages and then this becomes moot on the 'sending' side?

Reply:
Right about L49. I think that's sufficient.

As for the `audit_` namespace, I don't think it's needed anymore because of #126. To test, all we'd need to do is verify the activity appears in rabbitmq and rules fire via the NREP. Surprised I didn't include a test for that. We have this though - #128.

Reply:
See the conversation about continuation in PR #127.

Reply:
confirmed - not needed.  resolving.